### PR TITLE
fixed php warning and adjusted max_size to 10 mb.

### PR DESCRIPTION
**Pull request by bschilt**

### CODE REVIEW

File: src/Audit/Drupal/LargeDrupalFiles.php [hunk 1]
```diff
@@ -44,17 +44,17 @@ class LargeDrupalFiles extends Audit {
    * @inheritdoc
    */
   public function audit(Sandbox $sandbox) {
-    $max_size = (int) $sandbox->getParameter('max_size', 20000000);
+    $max_size = (int) $sandbox->getParameter('max_size', 10000000);
```
Comment (fiasco) [inline src/Audit/Drupal/LargeDrupalFiles.php:47]: The `default` value specified in `@Param` actual supersedes this value.

File: src/Audit/Drupal/LargeDrupalFiles.php [hunk 2]
```diff
@@ -44,17 +44,17 @@ class LargeDrupalFiles extends Audit {
...
     $sandbox->setParameter('readable_max_size', $max_size / 1000 / 1000 . ' MB');
     $query = "SELECT fm.uri, fm.filesize, (SELECT COUNT(*) FROM file_usage fu WHERE fu.fid = fm.fid) as 'usage' FROM file_managed fm WHERE fm.filesize >= @size ORDER BY fm.filesize DESC";
     $query = strtr($query, ['@size' => $max_size]);
     $output = $sandbox->drush()->sqlQuery($query);
-
+print_r($output);
```
Comment (fiasco) [inline src/Audit/Drupal/LargeDrupalFiles.php]: Wanna get rid of this?

File: src/Audit/Drupal/LargeDrupalFiles.php [hunk 3]
```diff
@@ -44,17 +44,17 @@ class LargeDrupalFiles extends Audit {
...
     if (empty($output)) {
       return TRUE;
     }
 
-    $records = explode("\n", $output);
+    $records = is_array($output) ? $output : explode("\n", $output);
```
Comment (fiasco) [inline src/Audit/Drupal/LargeDrupalFiles.php:57]: In what conditions does `$output` already return as an array?

File: src/Audit/Drupal/LargeDrupalFiles.php (hunk 2 above)
Comment (bschilt) [reply src/Audit/Drupal/LargeDrupalFiles.php]: doh!

File: src/Audit/Drupal/LargeDrupalFiles.php (hunk 1 above)
Comment (bschilt) [reply src/Audit/Drupal/LargeDrupalFiles.php:47]: I've updated the @param value

File: src/Audit/Drupal/LargeDrupalFiles.php (hunk 3 above)
Comment (bschilt) [reply src/Audit/Drupal/LargeDrupalFiles.php:57]: The $output is usually an array for both D7 and D8. However, during early testing months ago, the $output was a string.

**Comment by bschilt** [comment]
@fiasco this can be reviewed again.

**Pull request merged**